### ISSUE TITLE
Question : Message Status

### ISSUE THREAD

**Issue by aorfevre** [yagop/node-telegram-bot-api#687]
Hi everyone,

I have an event triggered on msg that triggers the creation of a new message.
For example => 

```
var msg = {
chat : _chat_id,
message_id : _new_message_id
}
```

Once I get the event, I create a new message as is :
```
options = {
    parse_mode: "HTML",
    disable_web_page_preview: true,
   reply_to_message_id : msg._new_message_id
}

bot.sendMessage(msg._chat_id,"tmp text" , options)
```

The output is perfect.

Now from time to time; my original message gets deleted for multiple cause automaticly (one of my other bots). 

I would like therefore to automaticly delete the message that I created.

QUESTION : 
-> Can we get the status of a specific chat/message_id as the message https://t.me/chat/message_id is supposed to be deleted.

If I can have a status like : "Message does not exist", I could easily delete the new one.
I don't want to try and edit the original message 

Thank you for support,

Regards,
Alexandre

**Comment by alexandercerutti** [comment]
So, wait a moment. Let me recap.
You send a message in a group (I suppose?) where multiple bots are inserted.
When you send a message, the first bot receives the message and sends another one (like `tmp text`).
Meanwhile, another bot deletes it (the message YOU sent).

What is not that clear to me is what you are asking for and what you mean with "status".

**Comment by aorfevre** [comment]
Well the process is as is :

1/ User enters a message : Message 1 
2/ Bot 1 : respond with "reply method"
3/ Bot 2 detect that the message is unappropriates and deletes Message 1

4/ I want to delete Message from bot one.
I could add to step 2 :
```
bot1.sendMessage(msg.chat.id,"response bot 1).then(function(){
    setTimeout(function(){ 
         // HERE I WANT TO GET THE STATUS OF MESSAGE 1 after 2 seconds for example.
    },2000) 
})
```

**Comment by alexandercerutti** [comment]
The fact is that, as [Telegram says](https://core.telegram.org/bots/faq#why-doesn-39t-my-bot-see-messages-from-other-bots)
> Bots talking to each other could potentially get stuck in unwelcome loops. To avoid this, we decided that bots will not be able to see messages from other bots regardless of mode.

So the second bot cannot delete first bot message. And the first bot cannot detect if the second bot deleted a message.

You should implement together the logic of checking the suitability of the message sent and the reply if suitable.

**Comment by aorfevre** [comment]
I have implemented a timer on my first bot.

The idea is that I wait 5 sec and check if the message exists.
If the initial exists (and 2nd bot has not deleted it); I can send my response.

If message does not exists; I do not post.
How do I check that message does not exists ? I tried the code bellow and it seems to be working. When message has been deleted; there is not data-telegram-post in HTML return. 

   ```
 request(_uri, {
      json: true
    }, (err, res, body) => {
      if (body.indexOf("data-telegram-post") === -1) {
```
